src/components/controlComponent.ts

```diff
@@ -0,0 +1,10 @@
+import { format } from "util";
+import { ExtensionInstance } from "../instance";
+import networkComponent from "./networkComponent";
+
+let self = ExtensionInstance.prototype
+let serverInstance = new networkComponent(self)
+
+self.networkComponent = serverInstance
+
+console.log(format("controlComponent :: Server instantiated at %s", self.context.extensionPath))
```

Comment:
Component files should export classes, variables should then (public or private) be set inside of this class.

Reply:
What should I rename this to if this is not a component? I cannot run a component as a Node.JS child process because it would essentially end up as declaring a class and then instantly constructing it, which there's no point of as it is impossible to reuse it.

Reply:
Well then, if this is not a component -- should I move it to modules?